src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-free-periods/crud/tutorial-free-period-form/tutorial-group-free-period-form.component.html

```diff
@@ -7,35 +7,128 @@
+                            id="startDate"
                             required
-                            [class.is-invalid]="isDateInvalid"
-                            (focus)="markDateAsTouched()"
-                            [value]="dateControl?.value | artemisDate: 'long-date' : false : undefined : true"
-                            [owlDateTimeTrigger]="date"
+                            [class.is-invalid]="isStartDateInvalid"
+                            (focus)="markStartDateAsTouched()"
+                            [value]="startDateControl?.value | artemisDate: 'long-date' : false : undefined : true"
+                            [owlDateTimeTrigger]="startDate"
                             readonly
                             class="form-control"
                         />
-                        <button type="button" class="btn btn-secondary" [owlDateTimeTrigger]="date"><fa-icon [icon]="faCalendarAlt" /></button>
-                        <owl-date-time [firstDayOfWeek]="1" [pickerType]="'calendar'" #date [pickerMode]="'dialog'" />
+                        <button type="button" class="btn btn-secondary" [owlDateTimeTrigger]="startDate"><fa-icon [icon]="faCalendarAlt" /></button>
+                        <owl-date-time [firstDayOfWeek]="1" [pickerType]="'calendar'" #startDate [pickerMode]="'dialog'" />
                     </div>
-                    @if (dateControl?.invalid && (dateControl?.dirty || dateControl?.touched)) {
+                    @if (startDateControl?.invalid && (startDateControl?.dirty || startDateControl?.touched) && startDateControl?.errors?.required) {
                         <div class="alert alert-danger">
-                            @if (dateControl?.errors?.required) {
-                                <div>
-                                    {{ 'artemisApp.forms.tutorialFreePeriodForm.dateInput.requiredDateError' | artemisTranslate }}
-                                </div>
-                            }
+                            {{ 'artemisApp.forms.tutorialFreePeriodForm.dateInput.requiredDateError' | artemisTranslate }}
                         </div>
                     }
                 </div>
+
+                @if (isFreePeriod) {
+                    <label class="form-label" for="endDate">{{ 'artemisApp.forms.tutorialFreePeriodForm.dateInput.labelEnd' | artemisTranslate }}</label>
+                    <div class="input-group">
+                        <input formControlName="endDate" [(ngModel)]="formData.endDate" required [owlDateTime]="endDate" class="d-none" />
+                        <input
+                            name="endDate"
+                            id="endDate"
+                            required
+                            [class.is-invalid]="isEndDateInvalid"
+                            (focus)="markEndDateAsTouched()"
+                            [value]="endDateControl?.value | artemisDate: 'long-date' : false : undefined : true"
```

Comment:
You should remove `[value]`, it's possible to adapt the date format with `OWL_DATE_TIME_FORMATS`